### PR TITLE
Added merge multiple that outputs an array

### PR DESCRIPTION
**Pull request by herecydev**

### CODE REVIEW

File: tests/merge-multiple-tests.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        entry: {
+          main: './server.js'
+        }
+      }
+    ];
+
+    assert.deepEqual(merge(a, b), result);
+  });
+}
+
```
Comment (bebraw) [inline tests/merge-multiple-tests.js]: Extra test case to add. `a` with `client` and `b` with `server` alone. This should yield output with both configurations.

Test also partials like `a`: `client`, `server` + `b`: `server`, `test`.

**Comment by herecydev** [comment]
Give me a little bit, ES lint has flagged all the line endings, so need to fix those up. Will update shortly.

File: tests/merge-multiple-tests.js (hunk 1 above)
Comment (herecydev) [inline tests/merge-multiple-tests.js]: Okay those tests have been added. I think they conform to what you were saying

**Comment by bebraw** [comment]
Document this at the README. Good to go after that. Great work. 👍

**Comment by herecydev** [comment]
I've documented this in the readme, let me know if something else needs adding to it.

**Comment by bebraw** [comment]
Published. Amazing work. Thanks. 👍

**Comment by herecydev** [comment]
You're most welcome

**Pull request merged**